etc/kayobe/kolla/config/nova/nova-comute-ironic.conf

```diff
@@ -0,0 +1,15 @@
+[DEFAULT]
+# Don't limit the number of concurrent builds for the nova ironic compute
+# service.
+max_concurrent_builds = 0
+
+# Provide a blank DHCP domain to agent
+dhcp_domain=
+
+[ironic]
+# Ramp up maximum retries to allow time for baremetal node reboot and switch configs
+api_max_retries = 720
```

Comment:
Does this mean the node is locked all that time? This should be separate from the deploy timeout - it's just the retries for a single API call.

Reply:
Not sure, I can check and get back to you. This was added as node deletion was putting nodes in ERROR state (provision was fine with default), and it seemed to be API timeout that this made go away and node deletions became reliable. I'll check if the node's locked the whole time with this so large.

Reply:
Actually this is the Red Hat BZ that pointed me towards increasing that, as we saw the same error of " Node <uuid> can not be updated while a state transition is in progress" , but only with delete->clean->available,  not with provision.